app/controllers/curation_concern/permissions_controller.rb

```diff
@@ -0,0 +1,19 @@
+module CurationConcern
+  class PermissionsController < ApplicationController
+    include Worthwhile::CurationConcernController
+    self.curation_concern_type = ActiveFedora::Base
+
+    def confirm
+    end
+
+    def copy_visibility
+      Sufia.queue.push(CopyVisibilityJob.new(params[:id]))
+      flash_message = 'Updating file permissions. This may take a few minutes. You may want to refresh your browser or return to this record later to see the updated file permissions.'
+      redirect_to Sufia::Engine.routes.url_helpers.generic_work_path(params[:id]), notice: flash_message
```

Comment:
@scherztc  I wonder if this could be [sufia, curation_concern] Similar to what @hortongn put in for the form.